spec/view_objects/result_filters/qualification_view_spec.rb

```diff
@@ -0,0 +1,100 @@
+require_relative "../../../app/view_objects/result_filters/qualification_view"
```

Comment:
do we need it...

maybe something we can do in config?

Reply:
We usually use `require "rails_helper"` but that loads the whole app. The class under test here is just a plain ruby object and doesn't need the whole app. So when we run the tests just for this class they run much faster if we just require the class under test.